tests/Unit/Elliptic/Initialization/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY "Test_EllipticInitialization")
+
+set(LIBRARY_SOURCES
```

Comment:
I'm confused, why are the tests added in the other commits but not built and run?

Reply:
I just didn’t want to edit the file in every commit.

Reply:
Every commit should compile and test the code it adds. Could you please make the necessary adjustments?

Reply:
fine, could you check the fixups quickly?

Reply:
Looks good, thanks!